queue_services/entity-digital-credentials/requirements.txt

```diff
@@ -24,3 +23,14 @@ git+https://github.com/bcgov/business-schemas.git@2.18.15#egg=registry_schemas
 git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
 git+https://github.com/bcgov/lear.git#egg=sql-versioning&subdirectory=python/common/sql-versioning
 git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+
+# fix due to conflicting versions from legal-api import
+protobuf==3.19.5
```

Comment:
Thanks! Was just running into this starting to look at the existing DBC queue service.